examples/measure.html

```diff
@@ -1,10 +1,12 @@
 ---
 template: example.html
 title: Measure example
-shortdesc: Example of using the ol.interaction.Draw interaction for creating simple measuring application.
+shortdesc: Example of using the ol.interaction.Draw interaction for creating a simple measuring application.
 docs: >
-  <p><i>NOTE: If use geodesic measures is not checked, measure is done in simple way on projected plane. Earth curvature is not taken into account</i></p>
-tags: "draw, edit, measure, vector"
+  <p><i>NOTE: Area and length calculations are done using ol.proj.getLength and ol.proj.getArea to get answers in meters or square meters at any latitude.
+  The ol.geom.Linestring.getLength method does not give a length in meters for EPSG:3857 or EPSG:4326 linestrings.
```

Comment:
With the proposed change, for UTM and BNG etc, the ol.geom methods and
proposed ol.proj methods will give identical results. If a user asks for
length between (0,0) and (1000,0) with such coordinates then an answer of
1000 is what is expected.
On 27 Oct 2015 12:00, "Peter Robins" <notifications@github.com> wrote:

> In examples/measure.html
> <https://github.com/openlayers/ol3/pull/4311#discussion_r43112226>:
>
> >  docs: >
> > -  <p><i>NOTE: If use geodesic measures is not checked, measure is done in simple way on projected plane. Earth curvature is not taken into account</i></p>
> > -tags: "draw, edit, measure, vector"
> > +  <p><i>NOTE: Area and length calculations are done using ol.proj.getLength and ol.proj.getArea to get answers in meters or square meters at any latitude.
> > +  The ol.geom.Linestring.getLength method does not give a length in meters for EPSG:3857 or EPSG:4326 linestrings.
>
> I think the existing comment needs changing (the whole point of most
> projections is that they do take earth curvature into account), though that
> can be done in a separate PR.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openlayers/ol3/pull/4311/files#r43112226>.
>